operator/internal/manifests/config.go

```diff
@@ -22,6 +21,12 @@ func LokiConfigMap(opt Options) (*corev1.ConfigMap, string, error) {
 		return nil, "", err
 	}
 
+	if opt.OpenShiftOptions.BuildOpts.OCPAlertManagerEnabled {
+		if err = ConfigureOptionsForMode(&cfg, opt.Stack.Tenants.Mode); err != nil {
+			return nil, "", err
+		}
+	}
```

Comment:
Can you flip the outer if block inside `openshift.ConfigureOptions(cfg)`. This way we limit exposing OpenShiftOptions outside of the openshift package.